pypi_formula_mappings.json

```diff
@@ -333,6 +333,10 @@
   "huggingface-cli": {
     "exclude_packages": ["certifi"]
   },
+  "iconsur": {
+    "package_name": "",
```

Comment:
why do we need this line?

Reply:
It's special `brew` logic I added to allow `brew update-python-resources iconsur` to work.

Once I finish https://github.com/Homebrew/brew/pull/16780, `bump`/`bump-formula-pr` will automatically update resources.

Reply:
nice, good to know 👍